pipeline_api/src/main/java/com/splicemachine/pipeline/constraint/UniqueConstraintChecker.java

```diff
@@ -48,7 +49,18 @@ public UniqueConstraintChecker(boolean isPrimaryKey,
     @Override
     public MutationStatus checkConstraint(KVPair mutation, DataResult existingRow) throws IOException {
         // There is an existing row, if this is an insert then fail.
```

Comment:
Update this comment with something similar to what's in UniqueConstraint.java please